3.10/ubuntu/Dockerfile

```diff
@@ -87,10 +92,11 @@ RUN set -eux; \
 	./Configure \
 		"$opensslMachine" \
 		enable-fips \
+		--prefix="$INSTALL_PATH_PREFIX" \
 		--openssldir="$OPENSSL_CONFIG_DIR" \
-		--libdir="/usr/local/lib/$debMultiarch" \
-# add -rpath to avoid conflicts between our OpenSSL's "libssl.so" and the libssl package by making sure /usr/local/lib/$debMultiarch is searched first (but only for Erlang/OpenSSL to avoid issues with other tools using libssl; https://github.com/docker-library/rabbitmq/issues/364)
-		-Wl,-rpath="/usr/local/lib/$debMultiarch" \
+		--libdir="$INSTALL_PATH_PREFIX/lib/$debMultiarch" \
```

Comment:
Can probably drop `$debMultiarch` here since we won't ever be doing multiarch in this path. :eyes: